orc8r/cloud/go/obsidian/swagger/combine.go

```diff
@@ -58,6 +58,26 @@
 	return combined, nil
 }
 
+// GetCombinedSpecFromService polls a service for its spec and combines
+// it with the common spec to return a combined spec.
+func GetCombinedSpecFromService(yamlCommon string, service string) (string, error) {
+	remoteSpec := NewRemoteSpec(service)
```

Comment:
Inline this into the next line

Reply:
This would actually require us to change the methods of RemoteSpec to use value receivers instead of pointer receivers? For the sake of uniformity, as it seems like most user defined types in the code base use pointer receivers, I'm thinking we should just ignore this one line change? Unless you have a strong preference to maybe switch over to value receivers for the methods, which I also understand the benefits of.

Reply:
From offline: is Go quirk, leave as-is